esp/templates/survey/review.html

```diff
@@ -34,9 +83,10 @@ <h1>Survey Responses for {{ program.niceName }} </h1>
 {% if perclass_data %}
 {% for c in perclass_data %}
     <table class="fullwidth">
-    <tr>
-        <th class="small">Responses for {{ c.class.emailcode }}: {{ c.class.title }}</th>
-    </tr>
+        {% ifchanged c.class.parent_class %}
+        <tr><th><h2><a name="class{{ c.class.parent_class.id }}"></a>Responses for {{ c.class.parent_class.emailcode }}: {{ c.class.title }}<span style="float:right;font-weight: normal;"><a href="#top">[to top]</a></span></h2></th></tr>
```

Comment:
Could the class name heading be larger than the section name heading? It feels weird to me that they are equal font sizes.